app/picks/WeeklyPicks.tsx

```diff
@@ -32,45 +32,49 @@ const FormSchema = z.object({
 
 export const revalidate = 900; // 15 minutes
 
-interface Props {
-  NFLTeams: Models.Document[];
-  currentGameWeek: IGameWeek;
-}
-
 /**
  * Renders the weekly picks page.
  * @param props - The page props.
  * @param props.NFLTeams - The NFL teams.
  * @param props.currentGameWeek - The current game week.
+ * @param props.leagueId - The league ID.
  * @returns {JSX.Element} The rendered weekly picks page.
  */
-const WeeklyPicks = ({ NFLTeams, currentGameWeek }: Props): JSX.Element => {
+const WeeklyPicks = ({
+  NFLTeams,
+  currentGameWeek,
+  leagueId,
+}: IWeeklyPicksProps): JSX.Element => {
   const [isLoaded, setIsLoaded] = useState<boolean>(false);
   const [userPick, setUserPick] = useState<string | null>(null);
-  const {
+  const { user, updateWeeklyPicks, weeklyPicks, gameWeek, updateGameWeek } =
+    useDataStore((state) => state);
+
+  const processGame = useProcessGame({
```

Comment:
There's no need to make this a const as there's no data being returned.

Reply:
@shashilo I think what is happening here is it is extracting the processGame function that is returned in the hook.  So it is grabbing processGame from the hook .... but I know you have comments to update the hook.  🤷🏻‍♂️